scripts/publish-packages.sh

```diff
@@ -0,0 +1,23 @@
+for d in */ ; do
```

Comment:
It would be nice if this script didn't depend on the current working directory being `packages`.

It could get its own path from `$0` and `cd` based on that.